### ISSUE TITLE
Investigate error in new summary code

### ISSUE THREAD

**Issue by 1beb** [Crunch-io/crunchtabs#153]
```
library(crunch)
login()

ds = loadDataset("https://app.crunch.io/dataset/3401649b145543aaaddf7d263c37f3bd")

# tabs banners
tabs_theme <- themeNew(
  default_theme = themeDefaultLatex()
)

tabvars <- c("q1", "q4", "q5", "q6", "q7_collapsed", "q8", "region", "marstat", "employ", "votereg", "newsint")
bnr_1 <- c("gender_bnr", "age4_bnr", "race_bnr")
bnr_2 <- c("educ_bnr", "income_bnr")
bnr_3 <- c("presvote16post_bnr", "pid3_bnr", "ideo_bnr")
crossbreaks <- banner(ds, list(bnr1 = bnr_1, bnr2 = bnr_2, bnr3 = bnr_3))
crosstabs <- crosstabs(ds, weight="weight_vote", vars=tabvars, banner=crossbreaks)
writeLatex(crosstabs,
           theme=tabs_theme,
           filename="tabs",
           pdf=TRUE,
           title="",
           sample="",
           field_period="",
           table_of_contents = TRUE,
           moe = "",
           subtitle="")
```

Crosstabs function appears to fail because of a naming issue. Could be related to hidden of otherwise changed values contained in the dataset. 

The error is: 

```
Error in names(var_types) <- vars : 
  'names' attribute [11] must be the same length as the vector [10]
In addition: There were 48 warnings (use warnings() to see them)
```

These are the offending codelines.
```
  var_types <- unlist(lapply(dataset[vars], class))
  names(var_types) <- vars
```
It appears that one

**Comment by 1beb** [comment]
It appears that someone has specified a variable in tabvars variable that exists in the dataset but is not otherwise accessible by crunch::tabbook. Adding an informative error message here would be useful. 

```
setdiff(names(ds), vars) # should be notthing)
```

**Comment by 1beb** [comment]
Has been fixed, will be merged with #152 updatest

**Comment by 1beb** [comment]
> I might go with the stop condition as well (because honestly if I'm in a big hurry I might miss a warning message). If possible, it might be cool if the error message can specify that the variable needs to be unhidden if it is in the dataset but just hidden (so as not to confuse the user).
